Modules/ThirdParty/DCMTK/CMakeLists.txt

```diff
@@ -93,7 +93,14 @@ if(ITK_USE_SYSTEM_DCMTK)
   set(ITKDCMTK_SYSTEM_INCLUDE_DIRS ${DCMTK_INCLUDE_DIRS})
 
   # Module standard library var
-  set(ITKDCMTK_LIBRARIES ${DCMTK_LIBRARIES})
+  set(ITKDCMTK_LIBRARIES)
```

Comment:
This seems like a longer code to accomplish the same thing. What is the benefit of this change?

Reply:
This allows to (1) conditionally set the variables `ITKDCMTK_LIBRARIES` with the list of DCMTK target names formatted as `DCMTK::NAME_OF_LIBRARY` and (2) support building against DCMTK version with/without support for exporting namespaced targets.